### PR TITLE
Support adapters in fixtures

### PR DESCRIPTION
**Pull request by wRAR**

### CODE REVIEW

File: docs/testing.rst [hunk 1]
```diff
@@ -49,3 +48,12 @@ to schedule requests or modify items returned from
...
 is set to ``True`` and using more simple logic in this case.
+
+.. _fixture-adapter:
+
+Configuring the item adapter
+============================
+
+As documented in :ref:`web-poet:web-poet-testing-adapters`, fixtures can use custom
+item adapters. ``savefixture`` will use the adapter specified in the
+``SCRAPY_POET_TESTS_ADAPTER`` setting when saving the fixture.
```
Comment (kmike) [inline docs/testing.rst:59]:
```suggestion
item adapters. ``savefixture`` command uses the adapter specified in the
``SCRAPY_POET_TESTS_ADAPTER`` setting to save the fixture.
```

File: tests/test_commands.py [hunk 2]
```diff
@@ -167,3 +164,59 @@ def to_item(self):
...
+            "savefixture",
+            type_name,
+            f"{server.root_url}",
+        )
+    fixtures_dir = cwd / "fixtures"
+    fixture_dir = fixtures_dir / type_name / "test-1"
+    fixture = Fixture(fixture_dir)
+    assert fixture.is_valid()
+    item = json.loads(fixture.output_path.read_bytes())
+    assert item == {"name": "chocolate"}
```
Comment (kmike) [inline tests/test_commands.py:222]: It seems we're not checking the item in other tests; would it make sense to check it, at least to ensure that the chocolate is not lower-case originally?

Comment (kmike) [reply tests/test_commands.py:222]: Alternatively, you can just return a constant from the page object, without using mock server.

File: docs/testing.rst (hunk 1 above)
Comment (Gallaecio) [reply docs/testing.rst:59]: Mind that, if you append “command”, you need to prepend an article as well: `savefixture` → _the_ `savefixture` command.

File: docs/settings.rst [hunk 3]
```diff
@@ -105,3 +105,14 @@ Default: ``fixtures``
...
 
 More info at :ref:`testing`.
+
+
+SCRAPY_POET_TESTS_ADAPTER
+-------------------------
+
+Default: ``None``
+
+Sets the class, or it's import path, that will be used as an adapter in the
```
Comment (Gallaecio) [inline docs/settings.rst:115]:
```suggestion
Sets the class, or its import path, that will be used as an adapter in the
```

**Review by Gallaecio — APPROVED** [review]

**Review by Gallaecio — APPROVED** [review]

**Pull request merged**